content/nextflow-on-aws-batch/40_nextflow202/31_od.md

```diff
@@ -1,13 +1,12 @@
 ---
-title: "On-Demand"
+title: "OD Compute Environment"
```

Comment:
_I propose to keep the acronyms to minimum, especially in titles, even if covered before (not everyone reads everything)._
**On Demand** Compute Environment

Reply:
@alexis-turpin-aws problem is that it creates a line-break within the menue... :(
Do you still reckon we use the long version?

Reply:
ok, deal